### PR TITLE
added force include hours countdown

### PR DESCRIPTION
**Pull request by FacuDeLorenzo**
Solves [ARBT-33](https://wakeuplabs.atlassian.net/browse/ARBT-33)

Just the functionality, will be styled later.
Also fixed hex string types

image

### CODE REVIEW

File: packages/ui/src/hooks/useArbitrumBridge.tsx [hunk 1]
```diff
@@ -52,10 +52,9 @@ export default function useArbitrumBridge() {
 
     // extract l2's tx hash first so we can check if this tx executed on l2 later.
     const l2Signer = await getSigner(childNetworkId);
-    const l2SignedTx = await inboxSdk.signChildTx(tx, l2Signer);
-    const l2Txhash = l2SignedTx;
+    const l2Txhash = (await inboxSdk.signChildTx(tx, l2Signer)) as `0x${string}`;
```
Comment (juandahl) [inline packages/ui/src/hooks/useArbitrumBridge.tsx:55]: I see you've used `0x${string}`. viem has this type as Address. it is maybe more readable for me, isn't it for you?

File: packages/ui/src/routes/activity.$tx.tsx [hunk 2]
```diff
@@ -16,7 +17,7 @@ import { formatEther } from "viem";
 
 export const Route = createFileRoute("/activity/$tx")({
   loader: async ({ params }) => {
-    const tx = transactionsStorageService.getByBridgeHash(params.tx ?? "0x");
+    const tx = transactionsStorageService.getByBridgeHash((params.tx) as `0x${string}` ?? "0x");
```
Comment (juandahl) [inline packages/ui/src/routes/activity.$tx.tsx:20]: maybe you can move the cast to the return of the function.

File: packages/ui/src/routes/activity.$tx.tsx [hunk 3]
```diff
@@ -93,7 +95,11 @@ function PostComponent() {
 
   useEffect(() => {
     setCanConfirm(!transaction.delayedInboxHash)
-  }, [transaction])
+    transaction.delayedInboxHash && getL1BlockTimestamp(transaction.delayedInboxHash).then(txTimestamp => {
+      const remaining = Math.floor(((txTimestamp + 24 * ONE_HOUR) - new Date().valueOf()) / ONE_HOUR);
```
Comment (juandahl) [inline packages/ui/src/routes/activity.$tx.tsx:99]: I dont know exactly the calcul but do you check if date-fns simplify it?

**Review by juandahl — APPROVED** [review]
LGTM. Just some comments to discuss 👍

File: packages/ui/src/hooks/useArbitrumBridge.tsx (hunk 1 above)
Comment (FacuDeLorenzo) [reply packages/ui/src/hooks/useArbitrumBridge.tsx:55]: It actually is! I'm changing it

File: packages/ui/src/routes/activity.$tx.tsx (hunk 2 above)
Comment (FacuDeLorenzo) [reply packages/ui/src/routes/activity.$tx.tsx:20]: It's actually being casted to be passed as parameter to the function!

File: packages/ui/src/routes/activity.$tx.tsx (hunk 3 above)
Comment (FacuDeLorenzo) [reply packages/ui/src/routes/activity.$tx.tsx:99]: Yes! It makes it much easier to use and read!

**Pull request merged**